researches/search.py

```diff
@@ -195,12 +199,24 @@ def get_web(parser: LexborHTMLParser) -> List[Web]:
                 title=title,
                 url=anchor,  # type: ignore
                 text=textof(item.css_first(".VwiC3b")),
-            )
+            ),
         )
 
     return items
 
 
+def get_lyrics(parser: LexborHTMLParser) -> List[Lyrics]:
+    items = []
+    for item in parser.css('div[jsname="U8S5sf"].ujudUb'):
```

Comment:
Don't use `jsname` and attributes. This may result in an unexpected stop for specific devices/platforms.